README.rst

```diff
@@ -322,6 +322,91 @@ subnet         Subnet UUID
 description    Optional description for the IP address
 ===========    ========================================
 
+Export Subnet View
+##################
+
+View to export subnet data.
+
+POST
+++++
+
+.. code-block:: text
+
+    /api/v1/subnet/<subnet-id>/export
+
+Import Subnet View
+##################
+
+View to import subnet data.
+
+POST
+++++
+
+.. code-block:: text
+
+    /api/v1/import-subnet
+
+------------
+
+Exporting and Importing Subnet
+==============================
+
+One can easily import and export `Subnet` data and it's Ip Addresses using `django-ipam`.
+
+Exporting
+#########
+
+Data can be exported via the admin interface or by using a management command. The exported data is in `.csv` file format.
+
+From management command
++++++++++++++++++++++++
+
+.. code-block:: shell
+
+    ./manage.py export_subnet <subnet value>
+
+This would export the subnet if it exists on the database.
+
+From admin interface
+++++++++++++++++++++
+
+Data can be exported from the admin interface by just clicking on the export button on the subnet's admin change view.
+
+.. image:: https://raw.githubusercontent.com/openwisp/django-ipam/master/docs/images/export.png
+
+Importing
+#########
+
+Data can be imported via the admin interface or by using a management command.
+The imported data file can be in `.csv`, `.xls` and `.xlsx` format.
+
+From management command
++++++++++++++++++++++++
+
+.. code-block:: shell
+
+    ./manage.py import_subnet --file=<file path>
+
+From admin interface
+++++++++++++++++++++
+
+Data can be imported from the admin interface by just clicking on the import button on the subnet view.
+
+.. image:: https://raw.githubusercontent.com/openwisp/django-ipam/master/docs/images/import.png
```

Comment:
is this screenshot included in the PR?

Reply:
Yes, I have added the files, - https://github.com/openwisp/django-ipam/pull/31/files#diff-6b41e55da1de1a57e6145c5989ca21db